lib/compress/zstd_compress.c

```diff
@@ -278,6 +304,13 @@ ZSTD_bounds ZSTD_cParam_getBounds(ZSTD_cParameter param)
     case ZSTD_p_ldmMinMatch:
     case ZSTD_p_ldmBucketSizeLog:
     case ZSTD_p_ldmHashEveryLog:
+
```

Comment:
This sets the bounds for all of these parameters to `0` and `1`.

Reply:
sorry, the function wasn't complete yet.
It is now.